tests/test_serialization.py

```diff
@@ -65,7 +65,7 @@ def test_load_from_invalid() -> None:
         ({"foo": 10, "bar": "${foo}"}, False, None, str),
         ({"foo": 10, "bar": "${foo}"}, False, None, pathlib.Path),
         ({"foo": 10, "bar": "${foo}"}, False, {"foo": 10, "bar": 10}, str),
-        ([u"שלום"], False, None, str),
+        (["שלום"], False, None, str),
```

Comment:
This has nothing to do with this commit, right?

Reply:
yes, it got auto fixed by something when I saved the file. (I think PyCharm).